service/src/main/kotlin/fi/espoo/evaka/attendance/RealtimeStaffAttendanceController.kt

```diff
@@ -101,7 +101,17 @@ class RealtimeStaffAttendanceController(private val accessControl: AccessControl
                                                 att.departedAutomatically
                                             )
                                         },
-                                plannedAttendances = plannedAttendances[employeeId] ?: emptyList()
+                                plannedAttendances = plannedAttendances[employeeId] ?: emptyList(),
+                                allowedToEdit =
+                                    accessControl
+                                        .checkPermissionFor(
```

Comment:
Running this in a loop can cause bad performance issues. Use the batch version of checkPermissionFor outside the loop (so you get a `Map<EmployeeId, AccessControlDecision>`) and lookup from the map instead